mindmeld/auto_annotator.py

```diff
@@ -835,5 +931,373 @@ def valid_entity_check(self, entity):
         return Entity.is_system_entity(entity) or entity in self.supported_entity_types
 
 
-register_annotator("SpacyAnnotator", SpacyAnnotator)
-register_annotator("BootstrapAnnotator", BootstrapAnnotator)
+class NoTranslationDucklingAnnotator(Annotator):
+    """Custom Annotator class used to generate annotations."""
+
+    def __init__(
+        self,
+        app_path,
+        annotation_rules=None,
+        language=None,
+        locale=None,
+        overwrite=False,
+        unannotate_supported_entities_only=True,
+        unannotation_rules=None,
+    ):
+        """Initializes a NoTranslationDucklingAnnotator.
+
+        Args:
+            app_path (str): The location of the MindMeld app.
+            annotation_rules (list): List of Annotation rules.
+            language (str, optional): Language as specified using a 639-1/2 code.
+            locale (str, optional): The locale representing the ISO 639-1 language code and \
+                ISO3166 alpha 2 country code separated by an underscore character.
+            overwrite (bool): Whether to overwrite existing annotations with conflicting spans.
+            unannotate_supported_entities_only (bool): Only allow removal of supported entities.
+            unannotation_rules (list): List of Annotation rules.
+        """
+        super().__init__(
+            app_path,
+            annotation_rules=annotation_rules,
+            language=language,
+            locale=locale,
+            overwrite=overwrite,
+            unannotate_supported_entities_only=unannotate_supported_entities_only,
+            unannotation_rules=unannotation_rules,
+        )
```

Comment:
You could do `super().__init__(*args, **kwargs)`

Reply:
Done